### PR TITLE
[#119372] Do not error on instrument validations when reservation interval is 0

### PR DESCRIPTION
**Pull request by jhanggi**
To reproduce:
* Go to Instruments#new
* In Reservation Restrictions, Leave the Interval (minutes) blank
* Put a value into either Minimum (minutes) or Maximum (minutes)
* Submit the form

The error was happening because reserve_interval was nil.

### CODE REVIEW

File: spec/models/instrument_spec.rb [hunk 1]
```diff
@@ -24,41 +24,58 @@
...
+    end
+
+    it 'does not error when minimum reservation is set, but reservation interval is not' do
+      instrument.reserve_interval = nil
+      instrument.min_reserve_mins = 30
+      expect { instrument.valid? }.not_to raise_error
+      expect(instrument.errors[:reserve_interval]).to be_present
+    end
+
+    it 'does not error when maximum reservation is set, but reserve_interval is not' do
```
Comment (drench) [inline spec/models/instrument_spec.rb]: Super minor, but it took me a bit to notice how this spec was different from the previous one. I'd consider doing this more in a setup-in-`context` style but then I'd feel the need to rework this whole file.

**Comment by drench** [comment]
One note on the spec style but that's a take or leave. :+1:

**Comment by jhanggi** [comment]
I've cleaned up the specs a bit, but only in that area. I'm leaving the rest of the file alone.

**Comment by drench** [comment]
:+1:

**Pull request closed without merging**